pages/services/cassandra/2.6.0-3.11.4/getting-started/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: layout.pug
+navigationTitle: Getting Started
+excerpt: Getting started with Cassandra
+title: Getting Started
+menuWeight: 20
+model: /services/cassandra/data.yml
+render: mustache
+---
+
+Getting started with a test instance of the DC/OS {{ model.techName }} service is straightforward.
+
+## Prerequisites
+
+- Depending on your security mode in Enterprise DC/OS, you may need to [provision a service account](/services/{{ model.serviceName }}/2.6.0-3.11.4/security/#provisioning-a-service-account) before installing. Only someone with `superuser` permission can create the service account.
+	- `strict` [security mode](/latest/security/ent/#security-modes) requires a service account.
+	- `permissive` security mode a service account is optional.
+	- `disabled` security mode does not require a service account.
+- Your cluster must have at least {{ model.install.minNodeCount }} private nodes.
+{{ model.install.customRequirements }}
+
+The default DC/OS Apache {{ model.techShortName }} installation provides reasonable defaults for trying out the service, but may not be sufficient for production use. You may require different configurations depending on the context of the deployment.
```

Comment:
<p class="message--note"><strong>NOTE: </strong>The default DC/OS Apache {{ model.techShortName }} installation provides reasonable defaults for trying out the service, but may not be sufficient for production use. </p>